File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.cpp
class Solution
{
public:
    vector<int> sortedSquares(vector<int> &nums)
    {
        vector<int> ans(nums.size());
        for (int i = 0; i < nums.size(); i++)
            ans[i] = nums[i] * nums[i];
        sort(ans.begin(), ans.end());
        return ans;
    }
};

File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.py
from typing import List


class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        for i in range(len(nums)):
            nums[i] = nums[i] ** 2    
        return sorted(nums)


nums = [-4, -1, 0, 3, 10]
print(Solution().sortedSquares(nums))
